database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -136,9 +136,6 @@ abstract class SlickDatabase(override val originalDatabaseConfig: Config) extend
 
   protected[this] lazy val insertBatchSize = databaseConfig.getOrElse("insert-batch-size", 2000)
 
-  protected[this] lazy val useSlickUpserts =
-    dataAccess.driver.capabilities.contains(JdbcCapabilities.insertOrUpdate)
```

Comment:
Somewhere there should be a comment explaining to future devs that Slick's MySQL Profile upserts are broken in 3.3.2, and should not be used until (insert Slick issue reference(s)) are fixed.